Specs/create-wishlist.feature

```diff
@@ -0,0 +1,31 @@
+Feature: Create wishlist
+
+    In order to remember the clothes I'd like to buy and/or share them with my friends
+    As an authenticated user
+    I create a wishlist
+
+    Scenario: Creating a wishlist
+        Given I am on "my profile page"
+        And I clicked on "create a wishlist"
+        And a "popin" appeared with a "title field" and some "preferences" like "share with friends or not"
+        When I "fill" the different fields
+        And I click on "Create this wishlist"
+        Then the wishlist is created
```

Comment:
Scenario: Creating a wishlist
 +        Given I am on "my profile page"
 +        And I clicked on "create a wishlist"
 +        When I "fill" the different fields
 +        And I click on "Create this wishlist"
 +        Then the wishlist is created
 +        And a "popin" appeared with a "title field" and some "preferences" like "share with friends or not"